int32 PERCENT_OUTPUT=0
int32 POSITION=1
int32 VELOCITY=2
int32 CURRENT=3
int32 FOLLOWER=5
int32 MOTION_PROFILE=6
int32 MOTION_MAGIC=7
int32 MOTION_PROFILE_ARC=10
int32 DISABLED=15

int32 mode 0
float64 value 0